i18n/translations/fr/common.json

```diff
@@ -48,6 +48,19 @@
     "regex": "Veuillez utiliser le bon format.",
     "file-size-too-large": "Sélectionner un fichier plus petit.",
     "file-size-too-large-all-files": "Les fichiers dépassent le maximum de {{maxSizeInMb}} Mo. Téléversez des fichiers plus petits.",
+    "file-upload": {
+      "default": {
```

Comment:
@anikbrazeau some new strings here

The default is for cases where we have an error but we don't know why